userspace/libsinsp/parsers.cpp

```diff
@@ -1688,7 +1697,7 @@ void sinsp_parser::parse_clone_exit(sinsp_evt *evt)
 		case PPME_SYSCALL_CLONE3_X:
 			parinfo = evt->get_param(14);
 			tinfo->set_cgroups(parinfo->m_val, parinfo->m_len);
-			m_inspector->m_container_manager.resolve_container(tinfo, m_inspector->is_live());
+			m_inspector->m_container_manager.resolve_container(tinfo, m_inspector->is_live() || m_inspector->is_syscall_plugin());
```

Comment:
`is_live()` and `is_syscall_plugin()` seem always paired, do you think we could introduce a common helper?

Reply:
I thought about it, but unfortunately they are not always paired: there are many cases in which we only want to use `is_live`, and others in which we want to pair `is_syscall_plugin` with `is_plugin`. I wanted to avoid adding many of these helpers, paying the tradeoff of making conditions like this more verbose. 😞